nodes/logger/logger.js

```diff
@@ -1,79 +1,195 @@
-module.exports = function (RED) {
-  const util = require('util');
-  const _ = require('lodash');
-  const fs = require('fs-extra');
-  const path = require('path');
+const util = require('util');
+const _ = require('lodash');
+const fs = require('fs-extra');
+const path = require('path');
 
+module.exports = function (RED) {
   function LoggerNode(config) {
     RED.nodes.createNode(this, config);
     const node = this;
     node.on('input', function (msg, send, done) {
```

Comment:
Bitte schreibe mir die Funktion um in eine asynchrone Funktion. Der Code wird übersichtlicher und dadurch fehlerunanfälliger. Verwende zudem die async-Funktionen der FS-Extra mit await.
```suggestion
    node.on('input', async (msg, send, done) => {
```